FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -237,6 +240,61 @@ - (void)callListeners:(NSString *)key config:(NSDictionary *)config {
   }
 }
 
+- (void)setCustomSignals:(nullable NSDictionary<NSString *, NSObject *> *)customSignals
```

Comment:
why not just take in `NSDictionary<NSString *, NSString *>`? then it becomes impossible to set something invalid

Reply:
Since we expect both NSString and NSNumber here, we cannot change this to accept only NSString values.